src/awx_plugins/credentials/azure_kv.py

```diff
@@ -1,11 +1,26 @@
-# FIXME: the following violations must be addressed gradually and unignored
-# mypy: disable-error-code="import-untyped, no-untyped-def"
+"""Microsoft Azure Key Vault Lookup Plugin.
+
+This module defines a credential lookup plugin to authenticate and retrieve
+secrets from an Azure Key Vault. If the Client ID, Tenant ID, and Client Secret
+are provided it will create a credential with those. If one is missing, it will attemmpt
+to use the Managed Identity of an Azure VM to create a credential.
+
+Functions:
+
+- :func:`azure_keyvault_backend`: Creates a credential either with the fields
+provided or via the VM environment, and retrieves the secret from the Key Vault.
+- ``azure_keyvault_plugin``: Defines the credential plugin interface.
```

Comment:
```console
 /home/docs/checkouts/readthedocs.org/user_builds/awx-plugins-core/checkouts/106/src/awx_plugins/credentials/azure_kv.py:docstring of awx_plugins.credentials.azure_kv:11: WARNING: Bullet list ends without a blank line; unexpected unindent. [docutils] 
```
*(https://app.readthedocs.org/projects/awx-plugins-core/builds/27888342/#267943005--42)*

I think, it wants this:
```suggestion
- ``azure_keyvault_plugin``: Defines the credential plugin interface.

```